wiki/Ranking_Criteria/id.md

```diff
@@ -35,6 +35,7 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
 
 ### Pedoman
 
+- **Pencipta dari segala media yang digunakan dalam beatmapset harus diberi kredit dalam deskripsi beatmapset.** Sesuai dengan [Pedoman Penggunaan Konten](/wiki/Rules/Content_Usage_Guidelines), media yang digunakan dalam beatmap harus diizinkan untuk digunakan di osu!, dan kebijakan lisensinya biasanya merinci mengenai cara memberi kredit kepada penciptanya. Alasan umum untuk menghilangkan kredit dari deskripsi yaitu sulitnya mencari sumber, keinginan pencipta untuk tidak diberi kredit, dan adanya kredit di kolom artis, judul, atau sumber.
```

Comment:
```suggestion
- **Pencipta dari segala media yang digunakan dalam beatmapset harus diberi pengakuan dalam deskripsi beatmapset.** Sesuai dengan [Pedoman Penggunaan Konten](/wiki/Rules/Content_Usage_Guidelines), media yang digunakan dalam beatmap harus diizinkan untuk digunakan di osu! dan kebijakan lisensinya biasanya merinci mengenai cara memberi pengakuan kepada penciptanya. Alasan umum untuk menghilangkan pengakuan dari deskripsi yaitu sulitnya mencari sumber, keinginan pencipta untuk tidak diberi pengakuan, dan adanya pengakuan di kolom artis, judul, atau sumber.
```
what do you think kredit means lmao

Reply:
tbh gw agak bingung itu mau pake apaan so...yeah, changed.

instance lain yang pake itu juga bahasanya dirubah biar lebih mudah dipahami.